e2e-commons/scripts/blocks.js

```diff
@@ -3,18 +3,36 @@ const { generateNewBlock } = require('../utils')
 
 const homeWeb3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8541'))
 const foreignWeb3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8542'))
-const {user} = require('../constants.json');
+const {user, blockGenerator} = require('../constants.json');
+
 homeWeb3.eth.accounts.wallet.add(user.privateKey)
 foreignWeb3.eth.accounts.wallet.add(user.privateKey)
+homeWeb3.eth.accounts.wallet.add(blockGenerator.privateKey)
+foreignWeb3.eth.accounts.wallet.add(blockGenerator.privateKey)
 
 function main() {
   setTimeout(async () => {
-    generateNewBlock(homeWeb3, user.address)
-    generateNewBlock(foreignWeb3, user.address)
+    generateNewBlock(homeWeb3, blockGenerator.address)
+    generateNewBlock(foreignWeb3, blockGenerator.address)
     main()
-  }, 5000)
+  }, 1000)
 }
 
+homeWeb3.eth.sendTransaction({
```

Comment:
Instead of performing transaction you can add the account to the genesis files: [chain.json](https://github.com/poanetwork/tokenbridge/blob/speed-ui-e2e/parity/chain.json) and [chain-foreign.json](https://github.com/poanetwork/tokenbridge/blob/speed-ui-e2e/parity/chain-foreign.json)

Reply:
Right. Done.